conans/test/functional/profiles/test_python_profiles.py

```diff
@@ -0,0 +1,58 @@
+import textwrap
+import unittest
+
+from conans.test.utils.tools import TestClient
+
+
+class PythonProfilesTest(unittest.TestCase):
+
+    def test_python_profile_create(self):
+        client = TestClient()
+        profile_content = textwrap.dedent("""
+            import os
+
+
+            settings = {
+                 "os": "Windows",
+                 "compiler": "Visual Studio",
+                 "compiler.version": "16",
+                }
+            options = {"shared": "2"}
+            env = {"pkg:env_var": "bar"}
+            cxxflags = os.environ.get('HOME', "")
```

Comment:
Use a custom defined env-var, not something that can be defined or not in the system, and then check it later in the test.